doctr/utils/multithreading.py

```diff
@@ -29,8 +30,8 @@ def multithread_exec(func: Callable[[Any], Any], seq: Iterable[Any], threads: Op
 
     threads = threads if isinstance(threads, int) else min(16, mp.cpu_count())
     # Single-thread
-    if threads < 2:
-        results = map(func, seq)
+    if threads < 2 or os.environ['DOCTR_CONCURRENCY_DISABLE'] == 'true':
```

Comment:
```suggestion
    if threads < 2 or os.environ.get('DOCTR_CONCURRENCY_DISABLE', "").upper() in ENV_VARS_TRUE_VALUES:
```

and use
```python3
from doctr.file_utils import ENV_VARS_TRUE_VALUES
```

Reply:
good catch looks better of course :)

Reply:
Mmmh, I'd say it would be better to disable the concurrency by passing a `threads` value inferior to 2

Relatively to docTR, this is a relatively low-level function. Also, I thought the purpose of this PR was to make docTR compatible with AWS. If that's the case, from what I understood, the only thing we have to be wary about is passing `threads < 2`. So technically, I'd argue there is nothing to change in this file.

But I understand that could be easier here, so if everyone's on board, I agree with Olivier's suggestion :+1:

Reply:
@frgfm Yes, I agree with you, it's better to handle this way: to propagate correctly the `threads` parameter.

- However, I just discovered that `threads` has not been passed in `doctr/models/preprocess/{pytorch, tensorflow}.py` so we need to propagate this parameter to higher levels -> another PR ?

- If we want to let enforce `thread=1` in AWS Lambda environment, according to [AWS Lambda Documentation about environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html), we can check that `AWS_LAMBDA_FUNCTION_NAME` is present in the environment variable.
So it should be:
```suggestion
    if threads < 2 or os.environ.get("AWS_LAMBDA_FUNCTION_NAME") is not None:
```

Reply:
This change is good in the sense user has less environment variables to configure, but every cloud platform I know offers Lambda-like service. And if another platform has similar security policy, it won't work (or user would have to manually set this variable, which is wierd). That's why I would stick to variable name like `DOCTR_DISABLE_MULTIPROCESSING`